src/lib/core/a11y/list-key-manager.ts

```diff
@@ -63,6 +64,9 @@ export class ListKeyManager<T extends CanDisable> {
       case END:
         this.setLastItemActive();
         break;
+      case ESCAPE:
```

Comment:
Hmm I see your point. We could have something like PopupKeyManager that inherits from the ListKeyManager instead.